src/common/language_dropdown.js

```diff
@@ -0,0 +1,14 @@
+import $ from 'jquery'
+
+export default class LanguageDropdown {
```

Comment:
:1st_place_medal: for using ES6.
This function can be moved inside the translator constructor and be used instead of the event listener like this:
```
$('ul#select_language li')
  .click(function onClick() {
     const lang = $(this).attr('class')
     document.location.search = `l=${lang}`
  })
....
```